Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocControlObject.cs

```diff
@@ -66,6 +66,25 @@ protected IReadOnlyList<string> GetValidationErrors ([NotNull] ElementScope scop
       if (IsReadOnly())
         throw AssertionExceptionUtility.CreateControlReadOnlyException();
 
+      return GetValidationErrorsInner (scope);
+    }
+
+    /// <summary>
+    /// Returns the validation errors for the read-only <param name="scope" />.
+    /// </summary>
+    /// <exception cref="MissingHtmlException">Thrown if the validation errors cannot be found due to faulty markup or missing validator.</exception>

```

Comment:
Exceptiontype, here and in line 62. Do we have other spots where the documented exception is now wrong?

Reply:
This is the only occurrence where the documentation is wrong after RM-6788. Should I fix this in this commit or a second one for RM-6788?

Reply:
separate commit. this here might get backported

Reply:
Wait. the exception doc is valid.
```csharp
private MissingHtmlException CreateMissingValidationErrorFieldException ()
{
  return new MissingHtmlException (
      $"Could not find validation error field of the control object with the ID '{GetHtmlID()}'. This could be due to wrong markup or a missing validator.");
}
```